File: UsingRoomORM/app/src/main/java/nisrulz/github/sample/usingroomorm/AppDatabase.java
package nisrulz.github.sample.usingroomorm;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {
    Person.class /*, AnotherEntityType.class, AThirdEntityType.class */
}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
  public abstract PersonDao getPersonDao();
}

File: UsingRoomORM/app/src/main/java/github/nisrulz/example/usingroomorm/AppDatabase.kt
package github.nisrulz.example.usingroomorm

import androidx.room.Database
import androidx.room.Room
import androidx.room.RoomDatabase

@Database(
    entities = [Person::class /*, AnotherEntityType::class, AThirdEntityType::class */],
    version = 1
)
abstract class AppDatabase : RoomDatabase() {
    abstract val personDao: PersonDao?
}